staging/src/k8s.io/api/core/v1/annotation_key_constants.go

```diff
@@ -95,6 +95,14 @@ const (
 	// configuration of a resource for use in a three way diff by UpdateApplyAnnotation.
 	LastAppliedConfigAnnotation = kubectlPrefix + "last-applied-configuration"
 
+	// defaultLogsContainerAnnotationName is an annotation name that can be used to preselect the interesting container
+	// from a pod when running kubectl logs.
+	DefaultLogsContainerAnnotationName = kubectlPrefix + "default-logs-container"
+
+	// defaultExecContainerAnnotationName is an annotation name that can be used to preselect the interesting container
+	// from a pod when running kubectl exec.
+	DefaultExecContainerAnnotationName = kubectlPrefix + "default-exec-container"
```

Comment:
Adding new annotations targeted at programming kubectl behavior is unexpected. Is there a KEP discussing this?

Reply:
not yet

Ok， I will open one later or raise it in the sig weekly/rugular meeting.

Reply:
@liggitt I opened https://github.com/kubernetes/enhancements/pull/2189 for discussion on `default container for pod`

I will update this pr once there's some conclusions.

Reply:
Add KEP kubernetes/enhancements#2189 and issue https://github.com/kubernetes/enhancements/issues/2227 to track this feature.